benchmarks/src/main/java/org/elasticsearch/benchmark/search/aggregations/AggConstructionContentionBenchmark.java

```diff
@@ -213,7 +213,7 @@ public MappedFieldType getFieldType(String path) {
         }
 
         @Override
-        public Collection<MappedFieldType> getMatchingFieldTypes(String pattern) {
+        public Set<String> getMatchingFieldNames(String pattern) {
```

Comment:
the aggregation context was not exposing this one. So I ended up having to add it. I do wonder why we don't just use SearchExecutionContext for aggs too.